tests/suites/test_suite_ctr_drbg.function

```diff
@@ -1,15 +1,96 @@
 /* BEGIN_HEADER */
+#include "mbedtls/entropy.h"
 #include "mbedtls/ctr_drbg.h"
 #include "string.h"
 
-static int test_offset_idx;
+/* Modes for ctr_drbg_validate */
+enum reseed_mode
+{
+    RESEED_NEVER, /* never reseed */
+    RESEED_FIRST, /* instantiate, reseed, generate, generate */
+    RESEED_SECOND, /* instantiate, generate, reseed, generate */
+    RESEED_ALWAYS /* prediction resistance, no explicit reseed */
+};
+
+static size_t test_offset_idx;
+static size_t test_max_idx;
```

Comment:
How about setting default values for `test_offset_idx` & `test_max_idx` so that the `if` will fail deterministically in case this function is called without setting values for these 2?

Reply:
I wrote the code to figure out how to map the official test data to our function calls. I didn't write it for robustness. Any robustness or readability improvement is appreciated.

Reply:
zeroed variables to make the test code more robust.